alembic/versions/8c6cddf6335d_add_panels_and_bands_plugins.py

```diff
@@ -0,0 +1,180 @@
+"""Add panels and bands plugins.
+
+Revision ID: 8c6cddf6335d
+Revises: 
+Create Date: 2016-05-11 15:26:23.703464
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '8c6cddf6335d'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+import sideboard
+import uber
+from uber import c
+
+def upgrade():
+    op.create_table('event',
+    sa.Column('id', sideboard.lib.sa.UUID(), nullable=False),
+    sa.Column('location', uber.models.Choice(c.EVENT_LOCATION_OPTS), nullable=False),
```

Comment:
Everywhere where you see a `c` option, that had to be added manually - alembic originally tried to add these columns with no Choice arguments! I don't know if that's easy to get around in a permanent sense.